Readme.md

```diff
@@ -191,6 +192,57 @@ while (keys.length) {
 
 [crockfordconvention]: http://javascript.crockford.com/code.html
 
+## Declare Classes
+
+Declare a class using the `@constructor` jsdoc annotation. Like,
+
+```js
+/**
+ * @constructor
+ */
+function SomeClass() {}
+```
+
+While there are several ways to attach methods and properties to an object
+created via `new`, the preferred style for methods is:
```

Comment:
Preferred and correct. If you do it any other way the method will be attached to the instance and not the prototype.